articles/hardware/hs64/workflow.md

```diff
@@ -0,0 +1,16 @@
+---
+uid: hs64_workflow
+title: Headstage64 Example Workflow
+---
+
+The example workflow below can by copy/pasted into the Bonsai editor using the clipboard icon in the top right. This workflow:
+- Captures data from the Bno055 IMU and passive probes via the RHD2164 amplifier, saving all data to disk.
```

Comment:
would invert the order because the main functionality of the headstage is probes, not IMU. Also, two separate bullets might convey this better.

- Captures data from passive probes via the RHD2164 amplifier, saving all data to disk.
- Captures data from the Bno055 IMU, saving all data to disk.

Note: make same change in neuropixels